ksqldb-engine/src/main/java/io/confluent/ksql/function/udaf/array/CollectListUdaf.java

```diff
@@ -47,54 +51,51 @@ private CollectListUdaf() {
   }
 
   @UdafFactory(description = "collect values of a Bigint field into a single Array")
+  public static <T> TableUdaf<T, List<T>, List<T>> createCollectListT() {
+    return new Collect<>();
+  }
+
   public static TableUdaf<Long, List<Long>, List<Long>> createCollectListLong() {
     return new Collect<>();
   }
 
-  @UdafFactory(description = "collect values of an Integer field into a single Array")
   public static TableUdaf<Integer, List<Integer>, List<Integer>> createCollectListInt() {
```

Comment:
These functions are unneeded now.  That said, they are public.  What do folks think about removing them?

Reply:
yeah we can remove them, nobody should be calling these directly